app/src/main/java/com/gatheringhallstudios/mhworlddatabase/features/monsters/detail/MonsterSummaryFragment.kt

```diff
@@ -145,12 +151,18 @@ class MonsterSummaryFragment : Fragment() {
         if (ailments.mud) addAilment(R.string.ailment_mud)
         if (ailments.effluvia) addAilment(R.string.ailment_effluvia)
 
-        if (ailmentList.isEmpty()) {
-            ailments_text.setText(R.string.general_none)
-            return
+        if (ailmentList.isEmpty()) addAilment(R.string.general_none)
```

Comment:
Doesn't this do nothing because you early exited earlier?

Reply:
We only exited if the `ailments` object was null. If we did all the specific ailment checks and the `ailmentList` is still empty, then we add a `none` string and push it to be displayed.